client/hosting/performance/utils.ts

```diff
@@ -6,3 +8,15 @@ export function isValidURL( url: string ) {
 		url
 	);
 }
+
+// Returns props for the support link
+export function getSupportLinkProps() {
+	return {
+		showIcon: false,
+		supportContext: isA8CForAgencies() ? 'a4a-site-performance' : 'site-performance',
+		showSupportModal: ! isA8CForAgencies(),
+		supportLink:
+			isA8CForAgencies() &&
+			'https://agencieshelp.automattic.com/knowledge-base/check-your-sites-performance',
+	};
+}
```

Comment:
This is to keep the env specific logic separate and not mix it with the components.